src/Codemirror.js

```diff
@@ -79,7 +82,7 @@ const CodeMirror = React.createClass({
 		);
 		return (
 			<div className={editorClassName}>
-				<textarea ref="textarea" name={this.props.path} defaultValue={this.props.value} autoComplete="off" />
+        <textarea ref={(node) => this.textarea = node} name={this.props.path} defaultValue={this.props.value} autoComplete="off" />
```

Comment:
Should also mention this. My app did not complain about `ref="textarea"` when I was using the npm package, but as soon as I ran `npm link` to my local copy, I started getting the "Only a ReactOwner can have refs" issue. My understanding is that this way of using refs is preferred anyway so hopefully it's not a problem to make this change.